checker/bin-devel/test-cftests-nonjunit.sh

```diff
@@ -17,3 +17,6 @@ source "$SCRIPTDIR"/clone-related.sh
 # Moved example-tests out of all tests because it fails in
 # the release script because the newest maven artifacts are not published yet.
 ./gradlew :checker:exampleTests -x javadoc -x allJavadoc --console=plain --warning-mode=all
+
```

Comment:
I think :checker:examplesTests are tests run makefiles https://github.com/eisop/checker-framework/blob/304e999c7c3ca727f9e1eec7349cb42cf79fe862/checker/build.gradle#L305

Reply:
Can you elaborate a bit what your point is?
We already test running `./gradlew clean` in `junit` above. Do you want to clean at the end of every test? Or why is `:checker:examplesTests` relevant for whether we call `clean` here?

Reply:
I think Junit test doesn't contain makefile tests such as `exampleTests`, is that correct? Because `exampleTests` are the tests for makefiles and `./gradlew clean` will also run make clean for makefile tests. I just want to make it works after we actual run makefile tests in CI.

Reply:
There are no tests that actually ensure that `clean` cleaned up everything - so you won't know whether this call to `clean` actually removed all files that were generated from `exampleTests`.
So what do you learn from calling `clean` here?

Reply:
okay, I will clean up them. Thanks!